src/java/test/config/findbugsExcludeFile.xml

```diff
@@ -144,4 +144,10 @@
     <Bug pattern="DM_DEFAULT_ENCODING" />
   </Match>
 
+  <!-- Disable 'Malicious code vulnerability warnings' due to mutable collection types in interface.
```

Comment:
We probably want to leave a note in ZOOKEEPER-1362 to remind ourselves to do it.